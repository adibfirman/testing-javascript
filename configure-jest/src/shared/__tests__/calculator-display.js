import React from 'react'
import {render} from 'react-testing-library'
import CalculatorDisplay from '../calculator-display'

it('should create a snapshot', () => {
  const {container} = render(<CalculatorDisplay value="0" />)
  expect(container.innerHTML).toMatchSnapshot()
})
