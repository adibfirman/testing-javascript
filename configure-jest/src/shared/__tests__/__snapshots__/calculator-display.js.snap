// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create a snapshot 1`] = `"<div class=\\"css-12149uf\\"><div class=\\"autoScalingText\\" style=\\"transform: scale(1,1);\\">0</div></div>"`;
